src/main/java/com/exactpro/th2/hand/builders/events/EventPayloadBuilder.java

```diff
@@ -59,6 +59,7 @@ public EventPayloadBuilder printTable(String tableHeader, Map<String, String> ta
 
 	public byte[] toByteArray() {
 		try {
+			//FIXME: use MAPPER from AbstractCommonFactory
 			return new ObjectMapper().writeValueAsBytes(this.data);
```

Comment:
I don't think that you need to use any mappers here. The Event wrapper from common library should work just fine, I believe